[INFO]2022-07-03 11:26:50,979: #################################################################
[INFO]2022-07-03 11:26:50,979: Perform log
[INFO]2022-07-03 11:26:50,979: #################################################################
[INFO]2022-07-03 11:26:50,980: name:56:Use case writer:zpou
[INFO]2022-07-03 11:26:50,982: Enter function : flow_weather()
[INFO]2022-07-03 11:26:50,982: Input parameter : update_dict, value : value is empty.
[INFO]2022-07-03 11:26:50,984: Enter function : flow_login()
[INFO]2022-07-03 11:26:50,984: Input parameter : update_dict, value : value is empty.
[ERROR]2022-07-03 11:26:50,986: apileft:82:name 'commen' is not defined
[ERROR]2022-07-03 11:26:50,987: apileft:83:Traceback (most recent call last):
  File "D:\my_api\tc_play\requetes.py", line 55, in apileft
    contex = eval(context)
  File "<string>", line 2, in <module>
NameError: name 'commen' is not defined

[ERROR]2022-07-03 11:26:50,988: post:61:local variable 'parm' referenced before assignment
[ERROR]2022-07-03 11:26:50,990: post:62:Traceback (most recent call last):
  File "D:\my_api\Public_variable\apitest.py", line 50, in post
    param = self.Requests.apileft(api, dict_request)
  File "D:\my_api\tc_play\requetes.py", line 85, in apileft
    return parm[0], parm[1], parm[2], url, headers, request_method
UnboundLocalError: local variable 'parm' referenced before assignment

[ERROR]2022-07-03 11:26:50,991: flow_login:29:local variable 'respones_reslut' referenced before assignment
[ERROR]2022-07-03 11:26:50,993: flow_login:30:Traceback (most recent call last):
  File "D:\my_api\confdb\flow\user\user.py", line 26, in flow_login
    login_result = self.api_login_byhy(login_dict)
  File "D:\my_api\confdb\api\user\user.py", line 19, in api_login_byhy
    return self.post(Tim_api_mgr_signin, request_dict)
  File "D:\my_api\Public_variable\apitest.py", line 64, in post
    return respones_reslut
UnboundLocalError: local variable 'respones_reslut' referenced before assignment

[INFO]2022-07-03 11:26:50,994: flow_weather:29:TypeError: update_dict is not dict!
[ERROR]2022-07-03 11:26:50,995: apileft:82:name 'commen' is not defined
[ERROR]2022-07-03 11:26:50,996: apileft:83:Traceback (most recent call last):
  File "D:\my_api\tc_play\requetes.py", line 55, in apileft
    contex = eval(context)
  File "<string>", line 2, in <module>
NameError: name 'commen' is not defined

[ERROR]2022-07-03 11:26:50,997: post:61:local variable 'parm' referenced before assignment
[ERROR]2022-07-03 11:26:50,998: post:62:Traceback (most recent call last):
  File "D:\my_api\Public_variable\apitest.py", line 50, in post
    param = self.Requests.apileft(api, dict_request)
  File "D:\my_api\tc_play\requetes.py", line 85, in apileft
    return parm[0], parm[1], parm[2], url, headers, request_method
UnboundLocalError: local variable 'parm' referenced before assignment

[ERROR]2022-07-03 11:26:51,000: flow_weather:35:local variable 'respones_reslut' referenced before assignment
[ERROR]2022-07-03 11:26:51,002: flow_weather:36:Traceback (most recent call last):
  File "D:\my_api\confdb\flow\user\client.py", line 30, in flow_weather
    weather_result = self.api_get_apone(weather_dict)
  File "D:\my_api\confdb\api\user\client.py", line 19, in api_get_apone
    return self.post(Tim_api_mgr_customers, request_dict)
  File "D:\my_api\Public_variable\apitest.py", line 64, in post
    return respones_reslut
UnboundLocalError: local variable 'respones_reslut' referenced before assignment

